__tests__/endpoints.test.js

```diff
@@ -113,6 +114,39 @@ test('GET: 200, responds with an array of all articles', ()=>{
 
 
 })
+
+test("GET: 200 respond with array of articles with a specific topics value", () => {
+
+    
+  return request(app)
+  
+    .get("/api/articles?topic=mitch")
+    .expect(200)
+    .then(({ body }) => {
+     const { articles } = body;
+     
+      expect(articles.length).toBe(12)
+      articles.forEach(article => {
+      expect(article.topic).toBe('mitch');
+      });
+      
+    });
+});
+
```

Comment:
One missing test here:
We have a topic which is valid, but has no associated articles (topic=paper)
In this case we would expect 200 responds with an empty array

Reply:
there is no paper. Topics in test data are mitch and cats. Don't understand what you mean by this